src/traceradvectiondiffusion.jl

```diff
@@ -266,4 +395,30 @@ function set_c!(prob, c)
   return nothing
 end
 
+"""
+    function set_c!(prob, c, nlayers)
+
+Set the initial condition for tracer concentration in all layers of a
+`TracerAdvectionDiffusion.Problem` that uses a `MultiLayerQG` flow to 
+advect the tracer.
+"""
+function set_c!(prob, c, nlayers)
+  sol, vars, grid = prob.sol, prob.vars, prob.grid
+
+  C = Array{Float64}(undef, grid.nx, grid.ny, nlayers)
```

Comment:
why do we need to define a new array here?
Is this different from what's in `vars.c`?

Reply:
The new array is to make the initial condition `c` the same size as `vars.c`. That means you can pass in `c` as an initial condition for one layer then the new array `C` copies `c` into the number of layers in the system. This is then the same size as `vars.c` so we can use the `MultiLayerQG.fwdtransform!` function. Do you think there neater way to do this?